pyproject.toml

```diff
@@ -48,4 +43,4 @@ exclude = [
 pythonVersion = "3.9"
 
 [tool.poetry.scripts]
-csvcubed = 'csvcubed.cli.entrypoint:entry_point'
\ No newline at end of file
+csvcubed = 'src.csvcubed.cli.entrypoint:entry_point'
```

Comment:
so we really need `src.` here? I thought setting the `from = "src"` above would be enough for it to realise that's where the entry points are.